client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -53,11 +56,18 @@ class TaxonomyManagerListItem extends Component {
 	};
 
 	render() {
-		const { onDelete, name, translate } = this.props;
+		const { onDelete, postCount, name, translate } = this.props;
 
 		return (
 			<div>
-				<span className="taxonomy-manager__label">{ name }</span>
+				<span className="taxonomy-manager__label">
+					{ name }
+				</span>
+				{ ! isUndefined( postCount ) &&
+					<span className="taxonomy-manager__count">
```

Comment:
Continuing the CSS comment: we may be able to avoid the wrapper here if you target `.count`.